fastlane/ScreenshotFastfile

```diff
@@ -53,7 +53,7 @@ platform :android do
     screenshot_devices = [
       {
         screenshot_type: 'phone',
-        device_name: 'Pixel_3_API_28',
+        device_name: 'Pixel_3_API_30',
```

Comment:
Note: Just checked (via `gcloud firebase test android models list`) the list of devices that Firebase Test Lab supports, and they don't yet seem to have the Pixel 3 API 30… 😒 

So if we merge this as is, we won't be able to follow the instructions (PCYsg-hal-p2) to generate the new screenshots from CI every time we want to re-generate them (e.g. when a new feature / screen design changes…)

Not sure why you had to update from API 28 to API 30 but since FTL doesn't have Pixel 3 API 30 (yet), maybe we should consider using another device here (one that is available in FTL too) to ensure the end result is the same / using the same device when run locally vs CI? 🤔 

<details><summary>List of devices available in FTL to date</summary>

```
┌───────────────────┬────────────────────┬──────────────────────────────────────┬──────────┬─────────────┬─────────────────────────┬───────────────┐
│      MODEL_ID     │        MAKE        │              MODEL_NAME              │   FORM   │  RESOLUTION │      OS_VERSION_IDS     │      TAGS     │
├───────────────────┼────────────────────┼──────────────────────────────────────┼──────────┼─────────────┼─────────────────────────┼───────────────┤
│ 602SO             │ Sony               │ Sony XZs                             │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ 801SO             │ Sony               │ Sony XZ3 801SO                       │ PHYSICAL │ 2880 x 1440 │ 28                      │               │
│ A0001             │ OnePlus            │ OnePlus One                          │ PHYSICAL │ 1920 x 1080 │ 22                      │               │
│ A1N_sprout        │ HMD Global         │ Nokia 8 Sirocco                      │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ ASUS_X00T_3       │ Asus               │ Asus ZenFone Max Pro M1              │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ ASUS_Z01KDA       │ Asus               │ Asus ZenFone 4                       │ PHYSICAL │ 1920 x 1080 │ 28                      │               │
│ B2N_sprout        │ HMD Global         │ Nokia 7 plus                         │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ DRG_sprout        │ HMD Global         │ Nokia 6.1 Plus                       │ PHYSICAL │ 2280 x 1080 │ 28                      │               │
│ F8331             │ Sony               │ Sony Xperia XZ F8331                 │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ F8332             │ Sony               │ Sony Xperia XZ                       │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ FRT               │ HMD Global         │ Nokia 1 (Android GO)                 │ PHYSICAL │  854 x 480  │ 27                      │               │
│ G8142             │ Sony               │ Sony XPERIA XZ Premium               │ PHYSICAL │ 1920 x 1080 │ 25,26                   │               │
│ G8232             │ Sony               │ Xperia XZs                           │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ G8342             │ Sony               │ Xperia XZ1                           │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ G8441             │ Sony               │ Xperia XZ1 Compact                   │ PHYSICAL │ 1280 x 720  │ 26                      │               │
│ H8216             │ Sony               │ Sony XZ2                             │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ H8266             │ Sony               │ Sony XZ2 Dual Sim                    │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ H8296             │ Sony               │ Xperia XZ2                           │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ H8314             │ Sony               │ Sony Xperia XZ2 Compact H8314        │ PHYSICAL │ 2160 x 1080 │ 26                      │               │
│ H8324             │ Sony               │ Xperia XZ2 Compact                   │ PHYSICAL │ 2160 x 1080 │ 26                      │               │
│ H8416             │ Sony               │ Sony XZ3                             │ PHYSICAL │ 2880 x 1440 │ 28                      │               │
│ HUR               │ SHARP              │ Sharp SHX39 KDDI                     │ PHYSICAL │ 2560 x 1440 │ 28                      │               │
│ HW-01K            │ Huawei             │ Huawei P20 Pro NTT Docomo            │ PHYSICAL │ 2240 x 1080 │ 27                      │               │
│ HWCOR             │ Huawei             │ Huawei Honor Play                    │ PHYSICAL │ 2340 x 1080 │ 27                      │               │
│ HWMHA             │ Huawei             │ Huawei Mate 9                        │ PHYSICAL │ 1920 x 1080 │ 24                      │               │
│ HWNEO             │ Huawei             │ Huawei Mate RS                       │ PHYSICAL │ 2880 x 1440 │ 28                      │               │
│ Nexus10           │ Samsung            │ Nexus 10                             │ VIRTUAL  │ 2560 x 1600 │ 19,21,22                │               │
│ Nexus4            │ LG                 │ Nexus 4                              │ VIRTUAL  │ 1280 x 768  │ 19,21,22                │               │
│ Nexus5            │ LG                 │ Nexus 5                              │ VIRTUAL  │ 1920 x 1080 │ 19,21,22,23             │               │
│ Nexus5X           │ LG                 │ Nexus 5X                             │ VIRTUAL  │ 1920 x 1080 │ 23,24,25,26             │               │
│ Nexus6            │ Motorola           │ Nexus 6                              │ VIRTUAL  │ 2560 x 1440 │ 21,22,23,24,25          │               │
│ Nexus6P           │ Google             │ Nexus 6P                             │ VIRTUAL  │ 2560 x 1440 │ 23,24,25,26,27          │               │
│ Nexus7            │ Asus               │ Nexus 7 (2012)                       │ VIRTUAL  │ 1280 x 800  │ 19,21,22                │               │
│ Nexus7_clone_16_9 │ Generic            │ Nexus7 clone, DVD 16:9 aspect ratio  │ VIRTUAL  │ 1280 x 720  │ 23,24,25,26             │ beta          │
│ Nexus9            │ HTC                │ Nexus 9                              │ VIRTUAL  │ 2048 x 1536 │ 21,22,23,24,25          │               │
│ NexusLowRes       │ Generic            │ Low-resolution MDPI phone            │ VIRTUAL  │  640 x 360  │ 23,24,25,26,27,28,29,30 │ beta=30       │
│ OnePlus3T         │ OnePlus            │ OnePlus 3T                           │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ OnePlus5          │ OnePlus            │ OnePlus 5                            │ PHYSICAL │ 1920 x 1080 │ 26                      │ deprecated=26 │
│ OnePlus5T         │ OnePlus            │ OnePlus 5T                           │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ OnePlus6T         │ OnePlus            │ OnePlus6T                            │ PHYSICAL │ 2340 x 1080 │ 28                      │               │
│ Pixel2            │ Google             │ Pixel 2                              │ VIRTUAL  │ 1920 x 1080 │ 26,27,28,29             │               │
│ SC-02K            │ Samsung            │ Samsung S9, SC-02K                   │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ SC-03K            │ Samsung            │ Samsung S9+, SC-03K                  │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ SH-03K            │ SHARP              │ Sharp Aquos R2 SH-03K                │ PHYSICAL │ 3040 x 1440 │ 28                      │               │
│ SO-01J            │ Sony               │ Sony Xperia XZ SO-01J                │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ SO-03J            │ Sony               │ Xperia XZ (Docomo, SO-03J)           │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ SOV33             │ Sony               │ Xperia X Performance SOV33 (JP)      │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ SOV34             │ Sony               │ Xperia XZ SOV34 (JP)                 │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ TB-8504F          │ Lenovo             │ Lenovo Tab 4 TB-8504F                │ PHYSICAL │ 1280 x 800  │ 27                      │               │
│ a5y17lte          │ Samsung            │ Galaxy A5 2017                       │ PHYSICAL │ 1920 x 1080 │ 24                      │               │
│ a9y18qlte         │ Samsung            │ A9 2018                              │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ a9y18qltechn      │ Samsung            │ Samsung SM-A9200                     │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ addison           │ Motorola           │ Moto Z Play Verizon XT1635-01        │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ albus             │ Motorola           │ Moto Z2 XT1710                       │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ anna              │ LG                 │ LG Q8 LGM-X800L                      │ PHYSICAL │ 2720 x 1440 │ 26                      │               │
│ astarqltechn      │ Samsung            │ Galaxy A8 Star                       │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ astarqlteskt      │ Samsung            │ Samsung A8 Star                      │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ athene            │ Motorola           │ Moto G4 Plus                         │ PHYSICAL │ 1920 x 1080 │ 23                      │               │
│ athene_f          │ Motorola           │ Moto G4                              │ PHYSICAL │ 1920 x 1080 │ 23                      │ deprecated=23 │
│ beyond1           │ Samsung            │ Galaxy S10 SM-G973F                  │ PHYSICAL │ 3040 x 1440 │ 28                      │               │
│ blueline          │ Google             │ Pixel 3                              │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ c5proltechn       │ Samsung            │ Samsung C5 Pro                       │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ capricorn         │ Xiaomi             │ Xiaomi MI 5s                         │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ crownlte          │ Samsung            │ Samsung Note 9                       │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ crownlteks        │ Samsung            │ Samsung Note 9 SM-N960N              │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ crownqlteue       │ Samsung            │ Galaxy Note 9 USA                    │ PHYSICAL │ 2960 x 1440 │ 27                      │               │
│ cruiserlteatt     │ Samsung            │ Samsung Galaxy S8 Active             │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ cruiserltesq      │ Samsung            │ Samsung S8 Active                    │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ deen_sprout       │ Motorola           │ Motorola One XT1941-4                │ PHYSICAL │ 1520 x 720  │ 28                      │               │
│ dipper            │ Xiaomi             │ Xiaomi MI 8                          │ PHYSICAL │ 2248 x 1080 │ 28                      │               │
│ dream2lte         │ Samsung            │ Samsung S8+ SM-G955F                 │ PHYSICAL │ 2960 x 1440 │ 26                      │               │
│ dream2lteks       │ Samsung            │ Galaxy S8+ SM-G955N                  │ PHYSICAL │ 2960 x 1440 │ 26                      │               │
│ dream2qltesq      │ Samsung            │ Samsung S8+ Verizon                  │ PHYSICAL │ 2960 x 1440 │ 26                      │               │
│ dream2qlteue      │ Samsung            │ Samsung S8+ SM-G955U1                │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ dreamqlteue       │ Samsung            │ Galaxy S8                            │ PHYSICAL │ 2960 x 1440 │ 26                      │               │
│ equuleus          │ Xiaomi             │ Xiaomi MI 8 Pro                      │ PHYSICAL │ 2248 x 1080 │ 28                      │               │
│ flame             │ Google             │ Pixel 4                              │ PHYSICAL │ 2280 x 1080 │ 29                      │               │
│ flo               │ Asus               │ Nexus 7 (2013)                       │ PHYSICAL │ 1920 x 1200 │ 19,21                   │               │
│ g3                │ LG                 │ LG G3                                │ PHYSICAL │ 2560 x 1440 │ 19                      │               │
│ grandpplte        │ Samsung            │ Galaxy J2 Prime SM-G532M             │ PHYSICAL │  960 x 540  │ 23                      │ deprecated=23 │
│ greatlte          │ Samsung            │ Samsung Galaxy Note 8 (SM-N950F)     │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ greatlteks        │ Samsung            │ Galaxy Note8 (N950N Korea)           │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ greatqlte         │ Samsung            │ Samsung Note 8                       │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ griffin           │ Motorola           │ Moto Z XT1650                        │ PHYSICAL │ 2560 x 1440 │ 24                      │               │
│ gts3lltevzw       │ Samsung            │ Samsung Tab S3 Verizon SM-T827V      │ PHYSICAL │ 2048 x 1536 │ 28                      │               │
│ gts4lltevzw       │ Samsung            │ Samsung SM-T837V                     │ PHYSICAL │ 2560 x 1600 │ 28                      │               │
│ gts4lvwifi        │ Samsung            │ Samsung Tab S5e SM-T720              │ PHYSICAL │ 2560 x 1600 │ 28                      │               │
│ h1                │ LG                 │ LG G5 LG-H831                        │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hammerhead        │ LG                 │ Nexus 5                              │ PHYSICAL │ 1920 x 1080 │ 21,23                   │               │
│ harpia            │ Motorola           │ Moto G Play (4th Gen) XT1607         │ PHYSICAL │ 1280 x 720  │ 23                      │               │
│ hero2lte          │ Samsung            │ Galaxy S7 edge                       │ PHYSICAL │ 1440 x 2560 │ 23                      │ deprecated=23 │
│ hero2qlteatt      │ Samsung            │ Samsung Galaxy S7 edge ATT           │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hero2qltechn      │ Samsung            │ Galaxy S7 SM-G9350                   │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hero2qltespr      │ Samsung            │ Samsung S7 Edge Sprint               │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hero2qltetmo      │ Samsung            │ Samsung Galaxy S7 edge T-Mobile      │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hero2qlteusc      │ Samsung            │ Galaxy S7 Edge SM-G935R4             │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hero2qltevzw      │ Samsung            │ Samsung Galaxy S7 edge Verizon G935V │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqlteaio       │ Samsung            │ Samsung S7 SM-G930AZ for Cricket     │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqlteatt       │ Samsung            │ Samsung S7 ATT                       │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ heroqltemtr       │ Samsung            │ Samsung S7 SM-G930T1                 │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqltespr       │ Samsung            │ Samsung S7 Sprint                    │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqltetfnvzw    │ Samsung            │ Galaxy S7 (Straight Talk) SM-G930VL  │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqltetmo       │ Samsung            │ Samsung S7 SM-G930T T-Mobile         │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqlteue        │ Samsung            │ Samsung S7 SM-G930U                  │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqlteusc       │ Samsung            │ Samsung S7 SM-G930R4                 │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ heroqltevzw       │ Samsung            │ Samsung S7 Verizon                   │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ htc_m8            │ HTC                │ HTC One (M8)                         │ PHYSICAL │ 1920 x 1080 │ 19                      │ deprecated=19 │
│ htc_ocedugl       │ HTC                │ HTC U Ultra Dual Sim                 │ PHYSICAL │ 2720 x 1440 │ 26                      │               │
│ htc_ocmdugl       │ HTC                │ HTC U11 plus                         │ PHYSICAL │ 2880 x 1440 │ 26                      │               │
│ htc_ocndugl       │ HTC                │ HTC U11                              │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ htc_pmeuhl        │ HTC                │ HTC 10                               │ PHYSICAL │ 2560 x 1440 │ 26                      │               │
│ hwALE-H           │ Huawei             │ Huawei P8 lite                       │ PHYSICAL │ 1280 x 720  │ 21                      │               │
│ james             │ Motorola           │ Moto E5 Play XT1921-2                │ PHYSICAL │ 1280 x 720  │ 26                      │               │
│ jeter             │ Motorola           │ Moto G6 Play XT1922-2                │ PHYSICAL │ 1440 x 720  │ 26                      │               │
│ joan              │ LG                 │ LG V30 LG-H932                       │ PHYSICAL │ 2880 x 1440 │ 26                      │               │
│ k61v1_basic_ref   │ Alps               │ TECNO KB8                            │ PHYSICAL │ 1500 x 720  │ 29                      │               │
│ lake_n            │ Motorola           │ Moto G7 Plus XT1965-2                │ PHYSICAL │ 2270 x 1080 │ 28                      │               │
│ lithium           │ Xiaomi             │ Xiaomi MIX                           │ PHYSICAL │ 2040 x 1080 │ 26                      │               │
│ lt02wifi          │ Samsung            │ Galaxy Tab 3                         │ PHYSICAL │  600 x 1024 │ 19                      │               │
│ lucye             │ LG                 │ LG G6 LGUS997                        │ PHYSICAL │ 2880 x 1440 │ 24                      │               │
│ lv0               │ LG                 │ LG K3                                │ PHYSICAL │  854 x 480  │ 23                      │               │
│ m0                │ Samsung            │ Samsung Galaxy S3                    │ PHYSICAL │ 1280 x 720  │ 18                      │               │
│ mata              │ Essential Products │ Essential PH-1                       │ PHYSICAL │ 2560 x 1312 │ 25                      │               │
│ mlv1              │ LG                 │ LG K4 (LG-X230)                      │ PHYSICAL │  854 x 480  │ 23                      │               │
│ natrium           │ Xiaomi             │ Xiaomi Mi 5s Plus                    │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ nitrogen          │ Xiaomi             │ Xiaomi Mi Max 3                      │ PHYSICAL │ 2160 x 1080 │ 28                      │               │
│ pettyl            │ Motorola           │ Motorola E5 play                     │ PHYSICAL │  960 x 480  │ 27                      │               │
│ platina           │ Xiaomi             │ Xiaomi Mi 8 Lite                     │ PHYSICAL │ 2280 x 1080 │ 28                      │               │
│ poseidonlteatt    │ Samsung            │ Samsung S7 Active ATT                │ PHYSICAL │ 1920 x 1080 │ 26                      │               │
│ sailfish          │ Google             │ Pixel                                │ PHYSICAL │ 1080 x 1920 │ 25,26,27,28             │               │
│ sawfish           │ Huawei             │ Huawei Watch 2                       │ PHYSICAL │  390 x 390  │ 26                      │ beta          │
│ seoul             │ Lenovo             │ Lenovo S5                            │ PHYSICAL │ 2160 x 1080 │ 26                      │               │
│ star2lte          │ Samsung            │ Samsung S9+ SM-G965F                 │ PHYSICAL │ 2960 x 1440 │ 28                      │               │
│ star2lteks        │ Samsung            │ Samsung S9+ SM-G965N                 │ PHYSICAL │ 2220 x 1080 │ 28                      │               │
│ star2qlteue       │ Samsung            │ Samsung Galaxy S9+ (US)              │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ starlte           │ Samsung            │ Samsung Galaxy S9 SM-G960F           │ PHYSICAL │ 2960 x 1440 │ 26                      │               │
│ starlteks         │ Samsung            │ Galaxy S9 (Korea)                    │ PHYSICAL │ 2220 x 1080 │ 28                      │               │
│ starqltechn       │ Samsung            │ Samsung Galaxy S9 SM-G9600           │ PHYSICAL │ 2960 x 1440 │ 26                      │               │
│ starqlteue        │ Samsung            │ Samsung Galaxy S9 (US)               │ PHYSICAL │ 2220 x 1080 │ 26                      │               │
│ tissot_sprout     │ Xiaomi             │ Xiaomi Mi A1                         │ PHYSICAL │ 1920 x 1080 │ 28                      │               │
│ tulip             │ Xiaomi             │ Redmi Note 6 Pro                     │ PHYSICAL │ 2280 x 1080 │ 28                      │               │
│ victara           │ Motorola           │ Moto X                               │ PHYSICAL │ 1920 x 1080 │ 19                      │               │
│ walleye           │ Google             │ Pixel 2                              │ PHYSICAL │ 1920 x 1080 │ 27,28                   │ default       │
│ zeroflte          │ Samsung            │ Galaxy S6                            │ PHYSICAL │ 2560 x 1440 │ 23                      │               │
└───────────────────┴────────────────────┴──────────────────────────────────────┴──────────┴─────────────┴─────────────────────────┴───────────────┘

```
</details>

Reply:
Ahh good catch @AliSoftware , I've thought about it a bit more and we might be able to keep these devices as 28, the original main reason was to just to run a later API but I don't think there's any big benefit to gain here

Reply:
Btw if we change the devices in `ScreenshotFastfile` here, we're supposed to also change that in 2 different places too if I'm not mistaken:

1. Update the relevant `fastlane/emulators/*.ini` (rename if needed and update the emulator config file with new API version etc) – so that when you run `bundle exec fastlane screenshots` to generate raw screenshots using Fastlane locally, it sets up the proper emulators before running the Snapshot Tests on them
2. Update the `.circleci/config.yml` file [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/.circleci/config.yml#L398-L401) and [here](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/.circleci/config.yml#L414) – so that when you run the screenshots on CI (which uses Firebase Test Lab to execute the unit tests) it uses the right emulators in FTL for it.

I see you reverted the changes and went back to API 28 so it's ok now 👍 , but I'm thinking maybe it could be worth a comment in the `ScreenshotFastfile` around this `screenshot_devices = …` line to mention it in case we're tempted to change the devices there in the future?